experimental/material_3_demo/lib/component_screen.dart

```diff
@@ -629,6 +633,117 @@ class _CheckboxRowState extends State<CheckboxRow> {
   }
 }
 
+enum Value { first, second }
+
+class Radios extends StatefulWidget {
+  const Radios({super.key});
+
+  @override
+  State<Radios> createState() => _RadiosState();
+}
+
+class _RadiosState extends State<Radios> {
+  Value? _value = Value.first;
+
+  @override
+  Widget build(BuildContext context) {
+    return Row(
+      mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+      children: <Widget>[
+        Radio<Value>(
+          value: Value.first,
+          groupValue: _value,
+          onChanged: (value) {
+            setState(() {
+              _value = value;
+            });
+          },
+        ),
+        Radio<Value>(
+          value: Value.second,
+          groupValue: _value,
+          onChanged: (value) {
+            setState(() {
+              _value = value;
+            });
+          },
+        ),
+      ],
+    );
+  }
+}
+
+class ProgressIndicators extends StatefulWidget {
+  const ProgressIndicators({super.key});
+
+  @override
+  State<ProgressIndicators> createState() => _ProgressIndicatorsState();
+}
+
+class _ProgressIndicatorsState extends State<ProgressIndicators> {
+  bool playCircularProgressIndicator = false;
+  bool playLinearProgressIndicator = false;
+
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+      children: <Widget>[
+        Row(
+          children: [
+            IconButton(
```

Comment:
Nice. I would just have one button for both of the indicators. Either they are both paused or both animating. That would be less cluttered.

Reply:
I see:)!